sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -570,21 +576,21 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
       throw new TypeError("The parameter 'onError' must be of type 'function'.");
     }
 
-    this._createMessageSessionIfDoesntExist()
-      .then(async () => {
-        if (!this._messageSession) {
-          return;
-        }
-        if (!this._isClosed) {
-          this._messageSession.receive(onMessage, onError, options);
-        } else {
-          await this._messageSession.close();
-        }
+    try {
+      if (!this._messageSession) {
```

Comment:
`_throwIfMessageSessionDoesntExist` would have taken care of this scenario, why check again?

Reply:
Removed